blech_clust.py

```diff
@@ -116,28 +123,64 @@
 #os.mkdir('memory_monitor_clustering')
 print('Created dirs in data folder')
 
-# Get the amplifier ports used
-ports = list(set(f[4] for f in file_list if f[:3] == 'amp'))
-# Sort the ports in alphabetical order
-ports.sort()
-
-## Pull out the digital input channels used, and convert them to integers
-#dig_in = list(set(f[11:13] for f in file_list if f[:9] == 'board-DIN'))
-#for i in range(len(dig_in)):
-#	dig_in[i] = int(dig_in[i][0])
-#dig_in.sort()
-
-# Read dig-in data
-# Pull out the digital input channels used, 
-# and convert them to integers
-dig_in_files = [x for x in file_list if "DIN" in x]
-dig_in = [x.split('-')[-1].split('.')[0] for x in dig_in_files]
-dig_in = sorted([int(x) for x in dig_in])
-
-# Read the amplifier sampling rate from info.rhd - 
-# look at Intan's website for structure of header files
-sampling_rate = np.fromfile('info.rhd', dtype = np.dtype('float32'))
-sampling_rate = int(sampling_rate[2])
+#Get lists of amplifier and digital input files
+if file_type == ['one file per signal type']:
+	electrodes_list = ['amplifier.dat']
+	dig_in_list = ['digitalin.dat']
+elif file_type == ['one file per channel']:
+	electrodes_list = [name for name in file_list if name.startswith('amp-')]
+	dig_in_list = [name for name in file_list if name.startswith('board-DI')]
+
+#Use info file for port list calculation
+info_file = np.fromfile(dir_name + '/info.rhd', dtype = np.dtype('float32'))
+sampling_rate = int(info_file[2])
+
+# Read the time.dat file for use in separating out the one file per signal type data
+num_recorded_samples = len(np.fromfile(dir_name + '/' + 'time.dat', dtype = np.dtype('float32')))
+total_recording_time = num_recorded_samples/sampling_rate #In seconds
+
+check_str = f'Amplifier files: {electrodes_list} \nSampling rate: {sampling_rate} Hz'\
+           f'\nDigital input files: {dig_in_list} \n ---------- \n \n'
+print(check_str)
+
+if file_type == ['one file per channel']:
```

Comment:
Pull port and dig-in info from info file
It already gets processed there

Reply:
Attempted this with [10706a6](https://github.com/abuzarmahmood/blech_clust/pull/38/commits/10706a65d85a827b502f929a116223683863f581) but dig-ins will still need to be read in blech_clust regardless